source/docs/forms/create_test_merchant_account.html.erb

```diff
@@ -38,6 +42,19 @@ breadcrumbs: false
   <%= text_field_tag :company_login, :placeholder => 'Your company', :id => 'company_login', :required => true %>
   </div>
 
+  <div class="form-group">
```

Comment:
Remove this control. this will be done in a separated ticket.

Reply:
The Kind of business combo code was removed